openpype/hosts/maya/plugins/publish/validate_rendersettings.py

```diff
@@ -305,6 +352,12 @@ def repair(cls, instance):
             "{aov_separator}", instance.data.get("aovSeparator", "_")
         )
 
+        for attribute, data in cls.get_nodes(instance, renderer).items():
+            if not data["values"]:
+                continue
+            for node in data["nodes"]:
+                lib.set_attribute(attribute, data["values"][0], node)
```

Comment:
Does the loop of the data["nodes"]do anything for the set_attribute function? Otherwise, this suggestion also applies although it is a bit with the ugly style(I know it's not quite readable).

```suggestion
            lib.set_attribute(attribute, data["values"][0],  data["nodes"][0])
```

Reply:
We need to set the attribute on all nodes, since there could (but unlikely) be multiple nodes.